### ISSUE TITLE
Regarding the Gaussian Flow loss guidance

### ISSUE THREAD

**Issue by archmelow** [Zerg-Overmind/diff-gaussian-rasterization#8]
Is it possible to backpropagate through pre-trained 4DGS model parameters with the given flow loss? I tried to implement it on 4D Gaussian Splatting, but the gradients to the deformation MLP network and the Gaussian parameters were all 0.
